test/main.js

```diff
@@ -31,6 +32,48 @@ var options = {
   templateDonut: {
     'twig': './templates/donut.twig'
   },
+  convertCategoryTitles: true,
+  convertCategoryTitlesData: {
+    "categories": {
+      "base": "00-atoms",
```

Comment:
These numbers are helping the unit test mimic the data in this file:

https://github.com/pattern-library/pattern-importer/blob/master/data/pattern-lab-categories.yml